untitled1/src/main/com/aca/homework/week7/clickable/AbstractClickable.java

```diff
@@ -0,0 +1,13 @@
+package com.aca.homework.week7.clickable;
+
+public abstract class AbstractClickable implements Clickable {
```

Comment:
because the name field is common, let's keep it here and implement the name() method here too